app/model/application.js

```diff
@@ -185,6 +197,26 @@ ApplicationSchema.statics.syncIssuesToGitHub = function(application) {
     .then(() => application);
 }
 
+ApplicationSchema.statics.syncLabelToGitHub = function(application) {
```

Comment:
also — i'm a bit confused about what this function is supposed to return. 

it seems like right now:

If the label exists, it will return a promise contain the result of GET request for the label.

If the label doesn't exist, it will return a promise containing the application data (with the new label set).

If the POST request fails, the function will return a promise containing the application data (with no label set).